src/Components/3DV/SceneView.tsx

```diff
@@ -592,33 +603,22 @@ function SceneView(props: ISceneViewProps, ref) {
         radius: number,
         meshes: AbstractMesh[],
         frames: number,
+        reset?: boolean,
         zoomOnly?: boolean
     ) => {
-        const positionFrom = cameraRef.current.position;
         const targetFrom = cameraRef.current.target;
         const radiusFrom = cameraRef.current.radius;
         // Now move it immediately to where we want it and save the new position
         cameraRef.current.zoomOn(meshes, true);
-        const positionTo = cameraRef.current.position;
-        const targetTo = cameraRef.current.target;
+        const targetTo = reset
+            ? initialCameraTargetRef.current
+            : cameraRef.current.target;
         const radiusTo = radius;
         // Reset camera back to original position
-        cameraRef.current.position = positionFrom;
         cameraRef.current.target = targetFrom;
         const ease = new BABYLON.CubicEase();
         ease.setEasingMode(BABYLON.EasingFunction.EASINGMODE_EASEINOUT);
         if (!zoomOnly) {
-            BABYLON.Animation.CreateAndStartAnimation(
```

Comment:
why remove only this one? I'm not following why we have multiple anyways, what are they each for? Can we add comments for the rest of us?

Reply:
We had three animations for moving the camera, radius, target and position. In testing, I have realised the position animation is not required.